server/command.go

```diff
@@ -86,7 +86,7 @@ const (
 	commandAdd    = "add"
 	commandDelete = "delete"
 	commandList   = "list"
-
+	featurePipeline = "pipeline"
```

Comment:
Declare it below `commandRun` and add a blank line to separate it out from other constants.